openfl-tutorials/interactive_api/Pytorch_Market_Re-ID/envoy_folder/start_envoy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+fx envoy start -n env_one --disable-tls -dh localhost -dp 50051
```

Comment:
```suggestion
fx envoy start -n env_one --disable-tls -dh localhost -dp 50051  -sc shard_config_one.yaml
```

With this key, it is obvious how to change the `Shard Descriptor` config.